pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -170,8 +170,8 @@
     private LocalZooKeeperConnectionService localZooKeeperConnectionProvider;
     private Compactor compactor;
 
-    private final ScheduledExecutorService executor = Executors.newScheduledThreadPool(20,
-            new DefaultThreadFactory("pulsar"));
```

Comment:
@hangc0276  would you please help changed this value as Matteo suggested? `Runtime.getRuntime().availableProcessors()`

Reply:
Thanks for your review, i will update the pr.